src/backend/gl/src/queue.rs

```diff
@@ -472,8 +472,35 @@ impl CommandQueue {
             com::Command::BindProgram(program) => unsafe {
                 self.share.context.UseProgram(program);
             }
-            com::Command::BindBlendSlot(slot, blend) => {
-                state::bind_blend_slot(&self.share.context, slot, &blend);
+            com::Command::BindBlendSlot(slot, ref blend) => {
+                state::bind_blend_slot(&self.share.context, slot, blend);
+            }
+            com::Command::BindAttribute(ref attribute, handle, stride) => unsafe {
+                let gl = &self.share.context;
+                gl.BindBuffer(gl::ARRAY_BUFFER, handle);
+                gl.VertexAttribPointer(attribute.location, attribute.size, attribute.format, gl::FALSE,
```

Comment:
There are several function variants of `VertexAttribPointer` (https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glVertexAttribPointer.xhtml)
Could you either fix it now or add a todo for later?

Reply:
Is it fine to simply assume `glVertexAttribIPointer` for all integer formats? Or is it possible in Vulkan to do something equivalent to calling `glVertexAttribPointer` with an integer format?

Reply:
> Is it fine to simply assume glVertexAttribIPointer for all integer formats

Yes, if you mean `Uint/Int` suffix for Formats.
For example, `Format::R8Uint` and `Format::R8Uscaled` only differ in the calling function but not in the parameters.